architecture.tex

```diff
@@ -10,7 +10,12 @@ \subsection{Basic Usage}
 % symbols, various ways to declare them
 
 The following statement
-imports all SymPy functions into the global Python namespace.
+imports all SymPy functions into the global Python namespace.\footnote{
+Nevertheless, it is best to avoid such wildcard import statements
```

Comment:
Maybe some note like "This is used here to improve readability of the examples."